internal/pkg/skuba/deployments/ssh/states.go

```diff
@@ -32,12 +32,12 @@ type Runner func(t *Target, data interface{}) error
 
 func (t *Target) Apply(data interface{}, states ...string) error {
 	for _, stateName := range states {
-		klog.V(1).Infof("=== applying state %s ===", stateName)
+		klog.Infof("=== applying state %s ===", stateName)
```

Comment:
not sure why we would do this? [1]

Reply:
Hmm, I also mixed with another bug. I wanted to make in one shot as a logging stuff.
This change comes from  https://bugzilla.suse.com/show_bug.cgi?id=1140526.
```
At Daimler we had to realize that a node bootstrap can take some long time for several task (i.e. depending on timeouts or internet speed).
While this shows as "nothing happens" - the admin gets nervous and is asking "is it broken? what is it doing etc.?
```

Reply:
I'm coming a little late to the party here so I will not hold anything up. but this should be it's own PR if it is fixing something separate. I would also say that if we use `info` we should use the `V()` function. If we want it to always show, then maybe it is not an info level. This is another reason it should be it's own PR to enable thought on just this topic. originally when we made this we went the same path kubeadm did. `fmt` was how we "wrote" things to the screen if we wanted it to always be seen (aka, a message to the user)... I feel like we should not diverge without a consensus. Really a RFC on this would be good IMO.

Reply:
This info is showing basic level of information that Skuba is working on.
What I am doing was trying to inform users that Skuba is doing something in a high level. Instead, muting for 5 min is not nice to users.  
Outputs are mixed with fmt and klog in Skua as you know. Klogs shows timestamp and filename, On the other hand, fmt does not show those. I think people want to see timestamps and files in screen when some process is running from Skuba cmds. kubadm has some use of fmt.printf and some use of log.Infof in kubeadm. IMO, `fmt.print` is speaking to Users like usage of Skuba or complete. klog.info is giving high level info. Additionally this stage info can give users what stage causes an error when there is an error.